.authors.yml

```diff
@@ -1,6 +1,6 @@
 - name: Michael Sarahan
   email: msarahan@continuum.io
-  num_commits: 1330
+  num_commits: 665
```

Comment:
https://github.com/regro/rever/commits/master fixes a bug that multiplies commits by the number of aliases someone has in mailmap

Reply:
Good idea!